### ISSUE TITLE
Tests don't show Suite/Full Description

### ISSUE THREAD

**Issue by jjmartin** [Microsoft/nodejstools#364]
In the wiki doc there is an image showing the tests looking like "Test Suite 1: Test 1"

![Test Explorer](https://github.com/Microsoft/nodejstools/wiki/Images/UnitTestsDiscoveryMocha.png)

But I can't seem to see the describe elements of my tests to show in the list. I've tried using the various group-by options in the exporer...

**Comment by jjmartin** [comment]
also - is there another forum/place to get general support on this tool, or to discuss with other users?

**Comment by mousetraps** [comment]
Have you tried the [latest dev build](https://github.com/microsoft/nodejstools/releases)? We've fixed #334 in that build, which seems similar to the issue you're running into. 

This is our primary feedback channel, and the quickest way to get a response. But bear in mind that the core team is pretty small, and we can't investigate every issue immediately. Our external contributors have also been super helpful in answering Qs (especially when they're running into the same issue), and we'll throw a discuss tag or a question tag on issues that lend themselves to it.

**Comment by jjmartin** [comment]
yes, i see the describe(s) above the it's in the test listing now

**Comment by jjmartin** [comment]
i have tests passing on the mocha commandline that are not passing in the vs runner - not sure what is up with that

**Comment by mousetraps** [comment]
Have you tried right clicking the failing test and debugging? Do you have any specific settings for mocha outside of VS?

**Comment by jjmartin** [comment]
yea... i am working it out - the first issue i hit was i run mocha from the commandline at the project dir level and the VS test seems to have the working directory in the ./test folder...  i used a package called app-root-path to resolve that.

I don't think i have any specific settings for mocha or the testing - both are just default settings afaik.

**Comment by jjmartin** [comment]
the last issue i was having was an async issue - the vs test was having a timing issue where the expect/assert was happening before the promises inside my UUT had completed.  I am guessing that mocha without visualstudio running it and inspecting it was fast enough that it got through the promises first, where the visual studio ones ran the stuff in a different order.

i resolved it so that the call back i was passing in to my UUT has the expects in it.

**Comment by elusive** [comment]
I am seeing the original symptom.  I have setup blank node js console project and tried to setup unit tests using default settings and am not able to get the test explorer to show the suite/describe text as a node in the tests tree.  I see my project and all tests.  The display is the same if I remove all suite/describe calls from my tests js.  I have tried setting different UI and different combinations of describe/it or suite/test but to no avail.  also applied the fix from other bug where I switched t.title to t.fullTitle() and now I can get all of my tests in the project root but now with names that are prefixed with the suite text but no structure still.
